src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+                    String by = task.substring(slash_sign + BY_AT_INDEX);
+                    if (by.contains("-")) {
+                        LocalDate date = LocalDate.parse(by);
+                        by = date.format(DateTimeFormatter.ofPattern("MMM d yyyy"));
+                    }
+                    tasks[index] = new Deadline(ddl_taskName, 'D', by);
+                    index = tl.addTaskMessage(index, tasks[index]);
+                    String textToAppend = "D | 0 | " + ddl_taskName + " | " + by;
+                    Storage.appendToFile(filePath, textToAppend);
+                } catch (StringIndexOutOfBoundsException e) {
+                    ui.deadlineIndexError();
+                } catch (FileNotFoundException e) {
+                    ui.fileErrorMessage();
+                } catch (IOException e) {
+                    ui.exceptionMessage();
+                    e.getMessage();
+                }
+            } else if (task.contains("event")) {
+                try {
+                    int slash_sign = task.indexOf("/");
+                    String event_taskName = task.substring(EVENT_INDEX, slash_sign - 1);
+                    String at = task.substring(slash_sign + BY_AT_INDEX);
+                    tasks[index] = new Event(event_taskName, 'E', at);
+                    index = tl.addTaskMessage(index, tasks[index]);
+                    String textToAppend = "E | 0 | " + event_taskName + " | " + at;
+                    storage.appendToFile(filePath, textToAppend);
+                } catch (StringIndexOutOfBoundsException e) {
+                    ui.eventIndexError();
+                } catch (FileNotFoundException e) {
+                    ui.fileErrorMessage();
+                } catch (IOException e) {
+                    ui.exceptionMessage();
+                    e.getMessage();
+                }
+            } else if (task.contains("delete")) {
+                try {
+                    int delete_index = Parser.parserToInteger(task.substring(DELETE_INDEX));
+                    Task delete_task = tasks[delete_index - 1];
+                    tl.removeTaskMessage(index, delete_task);
+                    tl.removeTask(delete_index, index, tasks);
+                }  catch (StringIndexOutOfBoundsException e) {
+                    ui.deleteIndexError();
+                }
+            } else if (task.contains("find")) {
+                String find_task = task.substring(FIND_INDEX);
+                tl.findTasks(index, find_task, tasks);
+            } else { ui.showErrorMessage(); }
+            task = b.nextLine();
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        new Duke(filePath).run();
```

Comment:
It would be good to name your Duke instance (the object) first. This line is doing at least two things.